doc/source/user_guide/groupby.rst

```diff
@@ -1245,7 +1249,7 @@ Control grouped column(s) placement with ``group_keys``
    group keys added to the result index. Previous versions of pandas would add
    the group keys only when the result from the applied function had a different
    index than the input. If ``group_keys`` is not specified, the group keys will
-   not be added for like-indexed outputs. In the future this behavior
+   not be added for like-indexed outputs. In the future, this behavior
```

Comment:
I think a comma for this introductory phrase is optional. I wouldn't oppose having the comma when adding new text, but I don't think we should be modifying things like this when it is just a matter of taste.